ts/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 120,
+  "singleQuote": true,
+  "useTabs": true
```

Comment:
Do we normally use tabs with TypeScript? I have no objection to switching to spaces.

Reply:
Yes, tabs are the normal Faithlife style.